doc-src/manual.tex

```diff
@@ -274,6 +277,38 @@ \section{Configuration file}
   \label{figure:example}
 \end{figure}
 
+The figure~\ref{figure:example} shows an example of configuration file
+that can used to skip the shebang line of shell scripts:
+when the first line of \mytt{.sh} files starts with \mytt{\#!},
+\headache{} does not modify that line and considers that the header must
+start at the second line.
+
+\begin{figure}
+%BEGIN LATEX
+\begin{center}
+%END LATEX
+\begin{boxedverbatim}
+# SWI Prolog file
+ | ".*\\.pl" -> frame open:"#"  line:"#" close:"#"
```

Comment:
I guess that the frame should use the `%` character instead since the third line of the example start with a `%`

Reply:
Yes, in prolog comments are introduced with `%`. `#` is only used in the shebang pattern to skip the first line.